config/env/default.js

```diff
@@ -24,5 +24,7 @@ module.exports = {
     port: 12202,
     token: process.env.GRAYLOG_TOKEN
   },
-  historyLength: 50 
+  historyLength: 50,
+  maxMessageLength: 1000,
```

Comment:
Could you explain why you chose to set these 2 vars in this config file?
Did you assume they both are environment-dependant? (which this config file is for)